SPECS/binutils/binutils.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Contains a linker, an assembler, and other tools
 Name:           binutils
-Version:        2.37
-Release:        5%{?dist}
+Version:        2.40
```

Comment:
I would be slightly iffy with upgrading `binutils`.
Lets do a full build and ensure things work as we expect them to.
PS: Not the `DELTA_BUILD=y` .. but the full build from scratch & verify with an LKG as well.

Reply:
full build triggered. https://dev.azure.com/mariner-org/mariner/_build/results?buildId=415761&view=results

Reply:
Doublng down on what Falak mentioned.  Can we do this with just patches?

Reply:
will check if all patches can be backported.

Reply:
raised a new PR with patches for 10 of the 13 CVEs https://github.com/microsoft/CBL-Mariner/pull/6176